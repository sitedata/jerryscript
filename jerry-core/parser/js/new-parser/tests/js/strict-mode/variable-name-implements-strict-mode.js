/* This test contains strict mode parser tests. */

'use strict'

implements
