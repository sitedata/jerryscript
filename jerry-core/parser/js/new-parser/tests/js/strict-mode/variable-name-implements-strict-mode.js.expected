
--- Script parsing start ---


--- Script parsing end ---

Parse error 'Identifier name is reserved in strict mode.' at line: 5 col: 1
